<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QWidget" name="detailsAutore">
    <widget class="QLabel" name="Autore_labelAfferenze">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Afferenze</string>
     </property>
    </widget>
    <widget class="QLabel" name="Autore_labelCognome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cognome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Autore_Nome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Autore_Cognome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Autore_labelNome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Autore_listAfferenze">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Autore_Identificativo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Autore_labelIdentificativo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identificativo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Autore_Indietro">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>560</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Indietro</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="detailsRivista">
    <widget class="QLabel" name="Rivista_labelEditore">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Editore</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rivista_Nome">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Rivista_labelVolume">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Rivista_Volume">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Rivista_labelData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rivista_Editore">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Rivista_labelNome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rivista_labelAcronimo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Acronimo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rivista_Acronimo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rivista_Data">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Rivista_Indietro">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>560</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Indietro</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="detailsArticolo">
    <widget class="QLabel" name="Articolo_labelPrezzo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>155</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prezzo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Articolo_Titolo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Articolo_labelTitolo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Titolo</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Articolo_NumeroPagine">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Articolo_labelNumeroPagine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero di pagine</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Articolo_Keywords">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Articolo_labelKeywords">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Keywords</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Articolo_Prezzo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Articolo_labelIdentificativo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Indentificativo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Articolo_Identificativo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Articolo_Indietro">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>560</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Indietro</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listAutoriArticoli">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>201</y>
       <width>201</width>
       <height>215</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="AutoriArticoli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>196</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Autori</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Articoli correlati</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="Articolo_labelPubblicatoPer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pubblicato per</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Articolo_PubblicatoPer">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>440</y>
       <width>205</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="detailsConferenza">
    <widget class="QLineEdit" name="Conferenza_Acronimo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="Conferenza_NumeroPartecipanti">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>430</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelNumeroPartecipanti">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero partecipanti</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Conferenza_Nome">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelLuogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Luogo</string>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelOrganizzatori">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Organizzatori</string>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelAcronimo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Acronimo</string>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>475</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Conferenza_Luogo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Conferenza_labelNome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Conferenza_Data">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>470</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Conferenza_Indietro">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>560</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Indietro</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Conferenza_listOrganizzatori">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
